### PR TITLE
BBEPAM-174: Add message type helper and service uuid in core

### PR DESCRIPTION
**Pull request by testinggithubsucks**

### CODE REVIEW

File: dictionary/message_type.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	}
+
+	return nil
+}
+
+func (m MessageTypeID) ToUint64() uint64 {
+	return uint64(m)
+}
+
+func MessageTypeFromInt64(id int64) (MessageTypeID, error) {
```
Comment (Mehdzor) [inline dictionary/message_type.go:47]: MakeMessageTypeInt64

File: dictionary/message_type.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+func (m MessageTypeID) CheckValidity() error {
+	_, exist := messageTypeIDToTypeName[m]
+	if !exist {
+		return fmt.Errorf("there is no type with this id %v", m)
+	}
+
+	return nil
+}
+
+func (m MessageTypeID) ToUint64() uint64 {
```
Comment (Mehdzor) [inline dictionary/message_type.go:43]: UInt64

File: dictionary/message_type.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	messageTypeID := MessageTypeID(id)
+	_, exist := messageTypeIDToTypeName[messageTypeID]
+	if !exist {
+		return 0, fmt.Errorf("there is no type with this id %v", id)
+	}
+
+	return messageTypeID, nil
+}
+
+func MessageTypeFromUint64(id uint64) (MessageTypeID, error) {
```
Comment (Mehdzor) [inline dictionary/message_type.go:61]: MakeMessageTypeUInt64

File: service_uuid/service_uuid.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	}
+
+	su.serviceUUIDMutex.Lock()
+	defer su.serviceUUIDMutex.Unlock()
+
+	su.uuid = uuid
+	return nil
+}
+
+func getUUIDFromEnv() (string, error) {
```
Comment (Mehdzor) [inline service_uuid/service_uuid.go:33]: UUIDFromEnvironment 

without get

File: dictionary/message_type.go (hunk 2 above)
Comment (testinggithubsucks) [reply dictionary/message_type.go:43]: ++

File: dictionary/message_type.go (hunk 3 above)
Comment (testinggithubsucks) [reply dictionary/message_type.go:61]: ++

File: service_uuid/service_uuid.go (hunk 4 above)
Comment (testinggithubsucks) [reply service_uuid/service_uuid.go:33]: ++

**Pull request merged**